src/synth/fluid_synth.h

```diff
@@ -204,7 +218,25 @@ void fluid_synth_process_event_queue(fluid_synth_t* synth);
 /*
  * misc
  */
-
 void fluid_synth_settings(fluid_settings_t* settings);
 
+
+/* extern declared fluid_synth_polymono.c */
+int fluid_synth_set_basic_channel_LOCAL(fluid_synth_t* synth, int basicchan,int mode, int val);
+/* extern declared in fluid_synth_mono.c */
+int fluid_synth_noteon_mono_LOCAL(fluid_synth_t* synth, int chan, int key, int vel);
+int fluid_synth_noteoff_mono_LOCAL(fluid_synth_t* synth, int chan, int key);
+int fluid_synth_noteon_monopoly_legato(fluid_synth_t* synth, int chan, int fromkey, int tokey, int vel);
+int fluid_synth_noteoff_monopoly(fluid_synth_t* synth, int chan, int key, char Mono);
+
+fluid_voice_t*
+fluid_synth_alloc_voice_LOCAL(fluid_synth_t* synth, fluid_sample_t* sample, int chan, int key, int vel, fluid_zone_range_t* zone_range);
+
+void fluid_channel_set_onenote_monolist(fluid_channel_t* chan, unsigned char key, unsigned char vel);
+void fluid_channel_clear_monolist(fluid_channel_t* chan);
+void fluid_channel_invalid_prev_note_staccato(fluid_channel_t* chan);
+void fluid_channel_cc_legato(fluid_channel_t* chan, int value);
+void fluid_channel_cc_breath_note_on_off(fluid_channel_t* chan, int value);
```

Comment:
All `fluid_channel_*()` functions should be moved to fluid_chan.c

Reply:
This will be moved.